app/src/main/java/com/example/imageEditor/Extension.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.imageEditor
```

Comment:
```suggestion
package com.example.imageEditor.Utils
```
Đổi tên ``Extension.kt `` thành ``GsonExt.kt``

Và move hết các file ``Constant.kt`` ``NetworkUtils.kt`` và ``GsonExt.kt`` vào package ``Utils`` này